_pages/en_US/uninstall-cfw.txt

```diff
@@ -26,7 +26,16 @@ Note that if you have any payload files other than `GodMode9.firm` in the `/luma
 
 ### Instructions
 
-#### Section I - Safety Test
+#### Section I - Checking for Flipnote
+1. Open system settings
+1. Click Internet
+1. Click DS Conncetion Settings
+1. If you see Flipnote, you'll need to copy the clean save, follow these steps: https://3ds.hacks.guide/installing-boot9strap-(fredtool)
```

Comment:
Run-on sentence, and assumes that people still have the clean DS connection settings somewhere.

Reply:
@lilyuwuu I thought that the save was recoverable after deleting it. I guess I can put steps below for getting the clean save.